### DISCUSSION TITLE
Heaving trouble rendering pages when entering via swipe

### DISCUSSION THREAD

**Discussion by justincaze** [neuefische/questions#820] [Web]
I currently have a problem linking the pages correctly.
I can display everything via swipe, but I would like to land directly on the page. So when I swipe to the left (on joined), I also want to get .../joined in the url and also back again.
This just won't work via routing and I'm slowly running out of ideas ...
This is the only method so far that at least shows me what I want:

![Bildschirmfoto 2024-03-13 um 16 24 42](https://github.com/neuefische/questions/assets/155940239/dc216f53-dc14-4384-8e6d-ac5b57910793)

**Comment by NikRoe** [comment]
Hi Timur,

could you send me a PR of this feature, so I can try it out on my end?

Thanks in advance
